BuildDockerImage.sh

```diff
@@ -132,7 +132,11 @@ get_Boost(){
   #rm boost-$1.tar.gz
 }
 
-
+# Check to see if the user is running the script as root/with sudo
+if [ $EUID -eq 0 ]; then
```

Comment:
It sounds so cold, can we give it a little bit nicer of a personality?  I can help think through it if you like...